docker-compose.yml

```diff
@@ -55,14 +67,19 @@ services:
     volumes:
       - ./config:/local/config
       - ./files:/local/bin/files
+    depends_on:
+      pg:
+        condition: service_healthy
+      redis-standalone:
```

Comment:
do we need healthy redis for this?

Reply:
Without a healthy Redis service, the application fails to start as it tries to connect to Redis at startup.